poly/src/polynomial/multivariate/mod.rs

```diff
@@ -164,3 +168,12 @@ impl Ord for SparseTerm {
         }
     }
 }
+
+/// multilinear utility: swap bits
+pub(crate) fn swap_bits(x: usize, a: usize, b: usize, n: usize) -> usize {
```

Comment:
Can you add some more comments for what this function is doing

Reply:
more comments added